packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md

```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
+
+Toast notifications must always have a close button so the user can dismiss them. You may also include an optional time-out to dismiss the alert automatically after a period of time. Eight (8) seconds is a recommended timeout. Alerts should not dismiss while the user is hovering over them.
 
 #### When not to use
 Do not use a toast alert when an event requires immediate action or for synchronous events like form validation.
 
-#### Behavior
-Transient toast alerts automatically dismiss after a period of time. Eight (8) seconds is a recommended timeout, but alerts should remain visible if the user is hovering over them.
-
 ### Stacking multiple alerts
 Multiple alerts will stack. The most recent alert should appear at the top and others will push down. When alerts are deleted, any space above will be filled by pushing remaining alerts up in the stack order.
 
-An Alert Group component allows a set of alerts to be properly positioned and stacked as shown below.
-
-![alert group](./img/alert-group.png)
-
-### Modal alerts
-When a condition occurs that requires immediate action, a modal message dialog is recommended.
+An [Alert Group](documentation/react/components/alertgroup) component allows a set of alerts to be properly positioned and stacked as shown below.
 
-![modal notification](./img/modal-notification.png)
-
-For more information about designing modal messages see the [message dialog design guidelines](/design-guidelines/usage-and-behavior/modal).
+![alert group](./img/toast-notification-multiple.png)
 
 ## Content guidelines
 Message text should be succinct and clearly state what the issue or problem is. Embedded links should navigate the user to the location where they can take action.
 
-Alert notifications present special challenges for accessibility. Screen readers will announce the arrival and message content of an incoming toast alert, but user focus will not be placed there. This will make it difficult for these users to directly act on the alerts itself. Therefore we recommend that any message containing actions be descriptive of how the action can be performed in the application. Messages that contain links should specify the location in text as well as including a simple text link. For example, a simple notification message might take the form:
+Toast alerts present special challenges for accessibility. Screen readers will announce the arrival and message content of an incoming toast alert, but user focus will not be placed there. This will make it difficult for these users to directly act on the alerts itself. Therefore, we recommend that any message containing actions be descriptive of how the can navigate to perform the action in the application. Messages that contain links should specify the location in text as well as including a simple text link. For example, a simple notification message might take the form:
```

Comment:
I'm thinking it could be useful to Add a sub header named "Accessibility considerations" to highlight these important remarks and make them more "visible"!

Reply:
Good idea. I made that change.